### PR TITLE
Add new feature in NATS module to 2201.4.0 Release Note

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
$subject
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

**Review by ThisaruGuruge — APPROVED** [review]
LGTM

File: downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
 - Added support for multiplexing in GraphQL subscriptions
 - Added support to access the GraphQL field information from resolvers
 
+##### `nats` package
+
+- Added support for NATS JetStream client, publishing and subscribing
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:78]:
```suggestion
- Added support for publishing and subscribing of the NATS JetStream client
```

Comment (ThisaruGuruge) [reply downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:78]: This seems not right. Maybe this?

```suggestion
- Added support for NATS JetStream client with publishing and subscribing
```

Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:78]: Shall we say, `Added support for the NATS JetStream client with publishing and subscribing functionalities`

Comment (aashikam) [reply downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:78]: Does it convey that `JetStream client` itself is entirely new

Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:78]: How about saying, `Added support for the new NATS JetStream client with publishing and subscribing functionalities`

Comment (aashikam) [reply downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md]: fixed please approve

**Pull request merged**